### ISSUE TITLE
SDDM theme not working

### ISSUE THREAD

**Issue by jerbaroo** [catppuccin/nix#229]
I have the global NixOS Catppuccin flag enabled: https://github.com/jerbaroo/nixos/blob/ecdfb7a2982d13f0ba2353f3fc9fa2287bea4ac3/system/theme.nix#L4

And SDDM is also enabled: https://github.com/jerbaroo/nixos/blob/ecdfb7a2982d13f0ba2353f3fc9fa2287bea4ac3/system/graphics.nix#L11

However SDDM looks the same as it does in the other issue about SDDM being broken (#192).

My `catppuccin-nix` version is: [d34a94a17c6ec4a0c4e24b3e4336ea504d021f6d](https://github.com/jerbaroo/nixos/blob/ecdfb7a2982d13f0ba2353f3fc9fa2287bea4ac3/flake.lock#L27) 

```
> cat /etc/sddm.conf
[General]
DefaultSession=
DisplayServer=wayland
HaltCommand=/run/current-system/systemd/bin/systemctl poweroff
Numlock=none
RebootCommand=/run/current-system/systemd/bin/systemctl reboot

[Theme]
Current=catppuccin-mocha
FacesDir=/run/current-system/sw/share/sddm/faces
ThemeDir=/run/current-system/sw/share/sddm/themes

[Users]
HideShells=/run/current-system/sw/bin/nologin
HideUsers=nixbld1,nixbld10,nixbld11,nixbld12,nixbld13,nixbld14,nixbld15,nixbld16,nixbld17,nixbld18,nixbld19,nixbld2,nixbld20,nixbld21,nixbld22,nixbld23,nixbld24,nixbld25,nixbld26,nixbld27,nixbld28,nixbld29,nixbld3,nixbld30,nixbld31,nixbld32,nixbld4,nixbld5,nixbld6,nixbld7,nixbld8,nixbld9
MaximumUid=30000

[Wayland]
CompositorCommand=/nix/store/xzx28k96shd3l886s676iq01ngjkwi8n-weston-13.0.1/bin/weston --shell=kiosk -c /nix/store/za8lq4j5zlrln8l8lzvyylb4z4ngcbnq-weston.ini
EnableHiDPI=true
SessionDir=/nix/store/wg9x90w2fiy75mdczp46959ykpj638p1-desktops/share/wayland-sessions

```

**Comment by isabelroses** [comment]
Can you test if https://github.com/catppuccin/nix/pull/230 fixes the issue?

**Comment by getchoo** [comment]
merged in b6c854508d8c03f3ff06bf658d12b0ae8052d7a5. thanks for reporting this!
